rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -19,6 +21,18 @@
  */
 public final class RxView {
   /**
+   * Create an observable of attach and detach events on {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult
+  public static Observable<ViewAttachEvent> attachEvents(View view) {
```

Comment:
Ehh idea didn't pan out (was thinking about generifying `ViewAttachEventOnSubscribe`). 

Fair point on the wrapper allocations, though. Thinking of the RxLifecycle use case, it would actually be perfect for that, so I'll add those siblings. Should only be one extra `*OnSubscribe` class, so nothing too crazy with code duplication.